### ISSUE TITLE
History: going back doesn't work right anymore

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#817]
https://twitter.com/geekdima/status/858403728562454533

I've been neglecting this, but yes, this is a thing.

**Comment by geekdima** [comment]
**Steps to reproduce:**
1. Go to your profile
2. Go to someone else profile
3. Use browser history to return yo tour profile. 

**Expected behavior:** 
Loading of my profile about page

**Result:**
Only the at `@` tag changed to the correct one, all the other info stays the same from the other profile.

**Comment by geekdima** [comment]
Can also reproduce on:
* Firefox 53.0
* Opera 44.0.2510.1449

**Comment by Stuntddude** [comment]
Another symptom of possibly the same problem:

1. Go to any blog post
2. Click "My Game" link
3. Use browser's back button

Results in frankensteining the blog post with your game's comments, as shown (blog post on top, comments from my game page below):

![screen shot 2017-05-17 at 8 16 32 am](https://cloud.githubusercontent.com/assets/6372134/26155912/5bbc32b2-3ada-11e7-9dce-a856e0faf971.png)

**Comment by povrazor** [comment]
Yeah the biggest issues with this seem to be related to editing. Pushing back on a post you are/can't edit.
